### PR TITLE
LPS-109415 Set default pagination mode for DDM Form Layout

### PR DESCRIPTION
**Pull request by carolmariaabb**

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:forms

**Comment by carolmariaabb** [comment]
ci:test:forms

**Comment by natocesarrego** [comment]
@carolmariaabb, in the next pull, can you please add the number of the Jira ticket on the commit message of a revert commit? For example, instead of  `Revert "LPS-107311 adding ...` you should do `LPS-109415 Revert "LPS-107311 adding ...`. This is a common pattern for the commit messages used to revert other commits.

Thanks. 🙂

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/DDMFormLayoutJSONDeserializer.java [hunk 1]
```diff
@@ -67,8 +68,16 @@ public DDMFormLayoutDeserializerDeserializeResponse deserialize(
...
+
+			String paginationMode = jsonObject.getString("paginationMode");
+
+			if (Validator.isNotNull(paginationMode)) {
+				setDDMFormLayoutPaginationMode(paginationMode, ddmFormLayout);
+			}
+			else {
+				setDDMFormLayoutPaginationMode(
+					DDMFormLayout.WIZARD_MODE, ddmFormLayout);
+			}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/DDMFormLayoutJSONDeserializer.java]: Simple and great solution, @carolmariaabb. Congratulations! 👏 🎉 👩‍🎨

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/86098. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**